Svc/FramingProtocol/FprimeProtocol.cpp

```diff
@@ -128,7 +128,8 @@ DeframingProtocol::DeframingStatus FprimeDeframing::deframe(Types::CircularBuffe
     buffer.setSize(size);
```

Comment:
Similar to the `frame` function, this check should be a assert but a early return with an error code.

Reply:
Agreed!